src/app/views/organizations/item.js

```diff
@@ -41,6 +41,9 @@ var OrganizationView = Backbone.View.extend({
 
   render: function() {
     console.log("Rendering organization", this.model);
+
+    $('title').text('The Detroit Ledger: ' + this.model.get('title') + ' grant data');
```

Comment:
Can we do the org name first, like "Kresge Foundation grant data - The Detroit Ledger